guides/common/modules/ports_prerequisites.adoc

```diff
@@ -9,7 +9,13 @@ The following tables indicate the destination port and the direction of network
 | 443 | TCP | HTTPS | Subscription Management Services (access.redhat.com) and connecting to the Red{nbsp}Hat CDN (cdn.redhat.com).
 |====
 
+ifeval::["{Build}" == "satellite"]
 Except in the case of a disconnected {Project}, {ProjectServer} needs access to the Red{nbsp}Hat CDN. For a list of IP addresses used by the Red{nbsp}Hat CDN (cdn.redhat.com), see the Knowledgebase article https://access.redhat.com/articles/1525183[Public CIDR Lists for Red Hat] on the Red{nbsp}Hat Customer Portal.
+endif::[]
+
+ifeval::["{Build}" == "foreman"]
+If you plan to use Red{nbsp}Hat services, {ProjectServer} needs access to the Red{nbsp}Hat CDN. For a list of IP addresses used by the Red{nbsp}Hat CDN (cdn.redhat.com), see the Knowledgebase article https://access.redhat.com/articles/1525183[Public CIDR Lists for Red Hat] on the Red{nbsp}Hat Customer Portal.
+endif::[]
```

Comment:
Here is a problem, we will have many (if not all) blocks which should be rendered for both "foreman" and "foreman-deb". Unfortunately, I've checked the asciidoctor codebase and the "ifeval" statement does not provide regular matching or globbing (like "foreman*"). Also there is no "else" keyword available. However this syntax does work:

ifeval::["{Build}" == "satellite"]
Downstream only thing.
endif::[]
ifeval::["{Build}" != "satellite"]
Upstream only thing for both CentOS and Debian.
endif::[]

Reply:
This, for me can just be absent from the Debian guide as they will never use Red Hat CDN for  their Foreman server will they? @lzap